### ISSUE TITLE
Error from fit_caviar and fit_finemap after updating to latest dsc 0.3.13

### ISSUE THREAD

**Issue by ZacZZZ** [stephenslab/susie-paper#5]
Hi Stephens lab,

I updated SoS, DSC and dscutils to the latest version.

And now I have error from running 
`dsc susie.dsc --target run_comparison -o toy_comparison -c 8`

```
Error in dscrutils::run_cmd(cmd) : 
  CAVIAR command failed (returned a non-zero exit status)
Calls: finemap_mcaviar ... lapply -> FUN -> single_core -> run_caviar -> <Anonymous>
```
```
Error in dscrutils::run_cmd(cmd) : 
  finemap command failed (returned a non-zero exit status)
Calls: finemap_mvar ... lapply -> FUN -> single_core -> run_finemap -> <Anonymous>
```
Are these version issues?
Any suggestions will be helpful.

Thank you,
Zac

**Comment by ZacZZZ** [comment]
The errors are gone with the `-s all ` skipping command you suggested, but it is just not reporting the running time.

**Comment by ZacZZZ** [comment]
But they are different errors then [stephenslab/dsc#211](https://github.com/stephenslab/dsc/issues/211). Do you think it might affect the pipeline on other datasets?

**Comment by gaow** [comment]
@ZacZZZ most of issues you run into so far are DSC / dscrutils version issues but the particular issue about does not read like a version issue. It seems to suggest `FINEMAP` and `CAVIAR` command failed. Were you able to run `FINEMAP` and `CAVIAR` commands in the terminal without an error? 

Yes `-s all` will use my existing results instead. But it does not mean you can run those commands fine. A good first place to start checking is to make sure those two executable programs work on your computer.

No promises and a fixed timeline for it, but we are in the process of updating the fine-mapping benchmark using latest version of DSC. That should remove most of the issues you have, when we get it done.

**Comment by ZacZZZ** [comment]
Both `finemap` and `CAVIAR` work on my command line with the example data provided by these two software. But I use the version in your Dockerfile. I will try update them later and let you know how it goes.

**Comment by gaow** [comment]
Defer it to #6

**Comment by gaow** [comment]
@ZacZZZ it turns out a regression bug that we now fixed in 0.4.0, if you `pip install dsc -U` on your desktop / cluster it should work even without using docker ...
